controllers/imageclusterinstall_controller.go

```diff
@@ -159,14 +160,27 @@ func (r *ImageClusterInstallReconciler) Reconcile(ctx context.Context, req ctrl.
 			log.WithError(updateErr).Error("failed to update ImageClusterInstall status")
 			return ctrl.Result{}, updateErr
 		}
-		return ctrl.Result{Requeue: true}, nil
+		return ctrl.Result{RequeueAfter: 10 * time.Second}, nil
```

Comment:
Why change this?

Reply:
In order not to Requeue immediately as this will create tons on logs

Reply:
Right, but I'm just making sure I understand. It's not actually directly related to this change, right?

Reply:
Sorry , yep was on the way , I can drop the change

Reply:
Is it maybe related to https://issues.redhat.com/browse/MGMT-18536?

Reply:
I think it was added when i looked on this ticket, let me remove this line :)